TheraLang.Web/ClientApp/src/app/app.module.ts

```diff
@@ -223,7 +222,8 @@ export function HttpLoaderFactory(http: HttpClient) {
         whitelistedDomains: ["localhost:5000"],
         blacklistedRoutes: []
       }
-    })
+    }),
+    SortablejsModule.forRoot({ animation: 400 })
```

Comment:
i remember when you showed me, animation was too slow can we increase it?

Reply:
It's already increased from 600